language/English/strings.po

```diff
@@ -12792,7 +12792,12 @@ msgctxt "#25009"
 msgid "The menu of this Blu-ray is not supported"
 msgstr ""
 
-#empty strings from id 25010 to 29799
+#: xbmc/video/dialogs/GUIDialogVideoBookmarks.cpp
+msgctxt "#25010"
```

Comment:
could you evaluate where 21396 is used and if it's still needed or could be changed to this format? Also a contextual note would be nice. Something like this maybe:
#. Used to build chapter names for media like videos, dvd, bluray, ...